src/status_im/contexts/communities/actions/invite_contacts/view.cljs

```diff
@@ -0,0 +1,133 @@
+(ns status-im.contexts.communities.actions.invite-contacts.view
+  (:require
+    [quo.core :as quo]
+    [quo.foundations.resources :as resources]
+    [quo.theme]
+    [react-native.core :as rn]
+    [react-native.gesture :as gesture]
+    [react-native.share :as share]
+    [status-im.common.contact-list-item.view :as contact-list-item]
+    [status-im.common.contact-list.view :as contact-list]
+    [status-im.contexts.communities.actions.invite-contacts.style :as style]
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]))
+
+(defn- no-contacts-view
+  [{:keys [theme id]}]
+  (fn []
```

Comment:
The component doesn't need to be a form-2 anymore. Btw, as we move away from Reagent atoms, form-2 components will be much rarer.